pkg/config/config.go

```diff
@@ -310,6 +310,7 @@ func initConfig(config Config) {
 	config.BindEnvAndSetDefault("jmx_thread_pool_size", 3)
 	config.BindEnvAndSetDefault("jmx_reconnection_thread_pool_size", 3)
 	config.BindEnvAndSetDefault("jmx_collection_timeout", 60)
+	config.BindEnvAndSetDefault("jmx_check_period", (15*time.Second)/time.Millisecond)
```

Comment:
Nit: this `15` should probably default to whatever `check.DefaultCheckInterval` is. I think if you just change it you'll get a dependency loop, so it might take some refactoring.

Reply:
We'll also want to document this in the config template and clarify this will impact the resolution granularity.

Reply:
@truthbk thanks for the review! In https://github.com/DataDog/datadog-agent/pull/3432/commits/9259722cbddc977571c65d28e02b6696d31b7267 I've refactored DefaultCheckInterval into a separate package. As you can see it increases the surface area of the PR a bit, but let me know if you're OK with the approach

Reply:
> We'll also want to document this in the config template and clarify this will impact the resolution granularity.

Where would I put this documentation?

Reply:
Apologies for the slow response, my bad! Got this merged now in the rebase PR. Thank you sir!